components/engine_rocks/src/import.rs

```diff
@@ -34,6 +34,19 @@ impl ImportExt for RocksEngine {
         Ok(())
     }
 
+    fn ingest_external_file_optimized(
```

Comment:
`ingest_external_file_optimized` actually is an implementation of `ingest_external_file_cf`, it is a historical problem.

Reply:
Currently importer use `ingest_external_file_cf`(maybe for performance consideration) but restore replica use `ingest_external_file_optimized`(maybe for not block online insert consideration), @Connor1996 can you confirm it? I hope we can only use one of them.

Reply:
Yes, seems `ingest_external_file_optimized` doesn't hurt the performance of importer, see https://github.com/tikv/rust-rocksdb/pull/218.
So let's just remove `ingest_external_file_optimized` from the trait and use `ingest_external_file_optimized` as implementation of `ingest_external_file_cf` in `engine_rocks`.

Reply:
great

Reply:
As mentioned in my other comment, I prefer not to introduce changes in behavior here. This is a separate problem that should be addressed separately.

Reply:
I filed a follow up issue https://github.com/tikv/tikv/issues/6162

Reply:
It's just forgotten to replace `ingest_external_file_cf` with `ingest_external_file_optimized`. It doesn't affect behavior. so just change it and make the trait clean?

Reply:
@brson Would it be possible to move the `ingest_external_file_optimized` logic up to the engine trait? The C or rust wrapper should not have real logic like these.